docs/source/conf.py

```diff
@@ -156,7 +156,7 @@ def setup(sphinx):
 TAGS = []
 smv_tag_whitelist = multiversion_regex_builder(TAGS)
 # Whitelist pattern for branches (set to None to ignore all branches)
-BRANCHES = ['master', 'v0.3', 'v1.0', 'v1.1']
+BRANCHES = ['master', 'v0.3', 'v1.0', 'v1.1', 'v1.2']
```

Comment:
isn't this only used from the `master` branch?

Reply:
yep, this change is meant for master. sorry